src/World.js

```diff
@@ -312,28 +313,31 @@ class World {
 	// accepts array of objects eg: [{sides:int, qty:int, mods:[]}]
 	// accepts object {sides:int, qty:int}
 	createNotationArray(notation){
+		const notation = Array.isArray( input ) ? input : [ input ]
```

Comment:
`const notation` is already defined by the function argument and `input` is not defined at all. Perhaps you meant
`notation = Array.isArray( notation ) ? notation : [ notation ]`?

Reply:
Fix. We're just ensuring the parameter coming in is in an array. Changed the parameter name to input.